Testing System/src/main/java/com/epam/practice/testingsystem/data/DataConnection.java

```
@@ -0,0 +1,21 @@
+package com.epam.practice.testingsystem.data;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+public class DataConnection {
+    public static Connection getConnection() {
+        String url = System.getenv("DB_URL");
+        String user = System.getenv("DB_USER");
+        String password = System.getenv("DB_PASSWORD");
+
+        try {
+            return DriverManager.getConnection(url, user, password);
+        }
+        catch (SQLException e) {
+            e.printStackTrace();
+            return null;
```

Please consider to throw an exception or do a retry for the connection instead of returning a null value. This could cause a serious threat in future development.

Please also consider to remove "return null" from the source code and replace it as much as you can with something meaningful. 

For example, if the method fails and it's critical for the application, you would probably need to throw a Runtime Exception (or derivatives like IllegalArgumentException) instead of proceeding the execution. If the method isn't that critical, consider finding workarounds for the situation.

 If none of the options are available you can use null value as a response from the method, but keep in mind that it could cause problems with further development.

---

Partially fixed. Now `null` values are used only as a response from `find(int id)` methods if there is no entry in the table with specified id. Otherwise a `RuntimeError` is thrown.